### ISSUE TITLE
Error: EXDEV: cross-device link not permitted

### ISSUE THREAD

**Issue by jperon** [Horta/linter-py#4]
With a /tmp mounted as tmpfs (on Ubuntu 16.04), linter-py raises this error : 

```
Error: EXDEV: cross-device link not permitted, link '/home/jacques/Programmes/Python/Bibliotheque/serveur.py' -> '/tmp/linter-py_13080Ux7QpMoEWeX4/Bibliotheque/serveur.py'
    at Error (native)
```

**Comment by davidlj95** [comment]
Had the same issue, I've temporarily solved the issue with this quick fix in a terminal

1. **Create a temporary folder in your home folder**
```bash
mkdir ~/tmp
```
2. **Export variable TMP to indicate temporary files folder**
```bash
export TMP=$HOME/tmp
```
3. **Launch Atom from terminal**
```bash
atom
```
Be careful with what you launch in the terminal now, as those applications may use that directory as temporal files directory too

**Comment by Horta** [comment]
Problem fixed with version 1.1.0 =)
